Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -518,7 +518,7 @@ test function with an instance of this trait type to control whether it runs:
   }
 }
 
-#### Converting XCTExpectFailure() calls
+#### Convert XCTExpectFailure() calls
```

Comment:
Delete; can't have subheadings within a section. Also, don't include symbols in headings.

Reply:
(And so on.)